### DISCUSSION TITLE
Poll for OpenCV 5.0 minimum required C++ standard

### DISCUSSION THREAD

**Discussion by s-trinh** [lagadic/visp#1336] [General]
Help the open-source community:
- by voting for the next OpenCV 5.0 minimum required C++ standard: https://github.com/opencv/opencv/discussions/25039
- sharing your ideas with https://github.com/opencv/opencv/issues/25007 and https://github.com/opencv/opencv/issues?q=is%3Aissue+is%3Aopen+label%3Aevolution

**Comment by fspindle** [comment]
@s-trinh Thanks, I didn't have seen that. I will vote for c++11!
